ProcessMaker/Cache/Settings/SettingCacheManager.php

```diff
@@ -3,15 +3,19 @@
 namespace ProcessMaker\Cache\Settings;
 
 use Illuminate\Cache\CacheManager;
+use Illuminate\Contracts\Cache\Repository;
+use Illuminate\Support\Facades\Redis;
 use ProcessMaker\Cache\CacheInterface;
 
 class SettingCacheManager implements CacheInterface
 {
-    protected CacheManager $cacheManager;
+    protected Repository $cacheManager;
 
     public function __construct(CacheManager $cacheManager)
     {
-        $this->cacheManager = $cacheManager;
+        $driver = env('CACHE_SETTING_DRIVER') ?? env('CACHE_DRIVER', 'redis');
```

Comment:
Use config instead of env.
In a **production** environment the **.env file is not loaded**, only the config from cache.

Reply:
It was fixed